test/lit.cfg

```diff
@@ -499,6 +499,7 @@ for target in config.llvm_code_generators:
 config.available_features.add("CPU=" + run_cpu)
 config.available_features.add("OS=" + run_os)
 config.available_features.add("PTRSIZE=" + run_ptrsize)
+config.available_features.add("VENDOR=" + run_vendor)
```

Comment:
Adding `VENDOR` is probably a good idea, but in the past we've usually used `REQUIRES: objc_interop` as a proxy for the Apple-ness of a platform.

Reply:
do you think there are cases where requiring one or the other would behave differently?